genon/preprocessor/module/intelligent_processor.py

```
@@ -0,0 +1,1675 @@
+                upload_tasks.append(asyncio.create_task(upload_files(file_list, request=request)))
+
+        if upload_tasks:
+            await asyncio.gather(*upload_tasks)
+
+        return vectors
+
+    def get_media_files(self, doc_items: list):
+        temp_list = []
+        for item in doc_items:
+            if isinstance(item, PictureItem):
+                name = path.rsplit("/", 1)[-1]
```

Ruff (0.15.10)

[error] Undefined name `path`

(F821)